src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double base;
+    private final double lateralSide;
+
+    IsoscelesTrapezoid(String color, double base, double lateralSide) {
+        super(color);
+
+        this.base = base;
+        this.lateralSide = lateralSide;
+    }
+
+    @Override
+    public double getArea() {
+        double firstLeg = base / 2;
+        double height = Math.pow(lateralSide, 2) - Math.pow(firstLeg, 2);
```

Comment:
To calculate the height, you need to take the square root of the result of the subtraction. Use `Math.sqrt(Math.pow(lateralSide, 2) - Math.pow(firstLeg, 2))` to calculate the height correctly.

Reply:
OK